README.md

```diff
@@ -52,7 +62,7 @@ Kjør tester
 Koble FP*-applikasjoner til VTP
 ------
```

Comment:
Litt inkonsekvent bruk av ## og ---- for overskrifter. 
---- gir også overskrift, men uklart hvilket nivå. Bruk helst bare en stil (##).

Reply:
Det er et bra tips